lib/ansible/modules/service_facts.py

```diff
@@ -318,6 +330,8 @@ def gather_services(self):
             if systemctl_path:
                 self._list_from_units(systemctl_path, services)
                 self._list_from_unit_files(systemctl_path, services)
+                self._list_from_units(systemctl_path, services, is_user=True)
```

Comment:
Could you please add a new option which controls this behaviour rather than enforcing it (to maintain backward compatibility)?

Reply:
Would it be possible to add an integration test for this change?

Reply:
OK, I'll add a new option `include_user` to control whether user services are included in facts, and try to add an integration test for the option.

Reply:
Executing a playbook as a regular user works correctly. However, when I tried to add an integration test and execute it in the testing container environment, I encountered an error: `Failed to connect to bus: No medium found`. This seems to stem from a conflict between running as `root` and executing `systemctl --user`. But after trying to switch to an unprivileged user using `ansible_become_user` and a pre-task `setup_test_user`, the problem persists. While this problem doesn't cause current tests to fail, it prevents fetching user services for further testing, which makes it probably unfeasible to design an integration test for the container environment. Any suggestions upon this issue?